index.js

```diff
@@ -194,6 +200,8 @@ function traverse(config) {
     config.visited[config.filename] = subTree;
   }
 
+  config.nonExistent = removeDups(config.nonExistent);
```

Comment:
We should do this here: https://github.com/dependents/node-dependency-tree/pull/49/files#diff-168726dbe96b3ce427e7fedce31bb0bcR73. We should only need to do this once before sending the result to the consumer.